2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,287 @@
+
+- family14, Trinity and Kabini removed from the master branch and moved to 4.18
+  branch
+- [boards
+  affected](https://doc.coreboot.org/releases/coreboot-4.19-relnotes.html#removed-mainboards):
+    * PC Engines apu1
+    * MSI MS-7721 (FM2-A75MA-E35)
+    * Lenovo AMD G505s
+    * HP Pavilion m6 1035dx
+    * ASUS F2A85-M (LE, PRO), A88XM-E and AM1I-A
+    * ASRock E350M1 and IMB-A180
+    * and others...
+- **Since then there were no AMD board removals yet**
+
+---
+
+# AMD coreboot status - last year
+
+- Starlabs could build coreboot firmware for their AMD laptops thanks to the
+  publication of Cezanne FSP to
+  [amd_blobs](https://github.com/coreboot/amd_blobs/tree/master/cezanne)
+  repository in September 2022
+    * [mb/starlabs/cezanne: Add Cezanne Byte Mk
+      I](https://review.coreboot.org/c/coreboot/+/69404/)
+    * [mb/starlabs/cezanne: Add Cezanne StarBook Mk VI
+      variant](https://review.coreboot.org/c/coreboot/+/68338/)
+    * **NO UPDATE TO THE PATCHES**
+- AMD Mendocino and Phoenix still in development with the former being in more
+  advanced state, FSP not published yet
+    * **FSP published for Mendocino, but not for Phoenix**
+- The FSP publication interval is quite long (1.5 a year between Picasso FSP
+  and Cezanne FSP release to public, and 1.25 a year after Cezanne APU release
+  FSP has been published)
+    * **Interval between Cezanne and Mendocino is only 5 months**
+    * **Mendocino is Zen2 while Cezanne is Zen3, so maybe not so big update**
+    * **Mendocino has been released to the market at the end of 2022**
+
+???
+
+Cezanne is newer core architecture, but Mendocino is a newer processor in
+terms of release date.
+
+---
+
+# AMD coreboot status - now
+
+- [patches covering KGPE-D16 bootblock support are out
+  there](https://review.coreboot.org/q/topic:kgpe-d16-bootblock)
+    * they have been abandoned because of lack of activity
+    * KGPE-D16 needs some love and attention, which, unfortunately, 3mdeb
+      can't humbly provide right now without any support from community
+- Marty Plummer ("hanetzer") is working on adapting AMD FSP for
+  Picasso/Cezanne to work on a non-Chromebook device ASRock x370 Killer SLI
+    * Join [Dasharo Matrix Space](https://matrix.to/#/#dasharo:matrix.org) or
+      [Dasharo vPubs](https://vpub.dasharo.com/o/1) to know more
+
+.center[.image-20[![](/img/ryzen.jpg)]]
+
+.center.footnote[
+[Ryzen photo](https://www.flickr.com/photos/130561288@N04/51371399197)
+by Fritzchens Fritz, CC0 1.0 Universal Public Domain Dedication
+]
+
+???
+
+hanetzer is putting some serous efforts here using SerialICE to trace the
+hardware accesses in FSP and PSP. Observes hangs around accesses to PSP and
+SMU.
+
+- PSP binaries: Picasso's work, but Cezanne (and maybe others) do not work
+  with anything but soldered memory and ChromeOS type devices
+- FSP binaries: hardcoded CPUID checks for the mobile variants, which need
+  patching or a wholly different compilation parameters.
```

Comment:
It is worth mentioning that during the last vPub, we decided that the best way forward would be to invest in OpenSSL.

Reply:
Ok, I see you talk about that more extensively below, but at least let's link things.

Reply:
Sure, I can put that in notes.

Reply:
https://github.com/3mdeb/conferences/pull/20/commits/f5e1054cc7e3e0347b1c5042cac0de78ecd32930